smcroute.service.in

```diff
@@ -8,8 +8,9 @@ After=network-online.target
 Requires=network-online.target
 
 [Service]
-Type=notify
-ExecStart=@SBINDIR@/smcrouted -n -s
+Type=simple
```

Comment:
One note here, when do 'sudo service smcroute restart',  it would always fail out of timeout.  'simple' type works well to me.

> Apr 14 08:10:14 raspberrypi systemd[1]: smcroute.service: Start operation timed out. Terminating.
> Apr 14 08:10:14 raspberrypi smcroute[28240]: Exiting.
> Apr 14 08:10:14 raspberrypi systemd[1]: smcroute.service: Unit entered failed state.
> Apr 14 08:10:14 raspberrypi systemd[1]: smcroute.service: Failed with result 'timeout'.

@troglobit , any thoughts?

Reply:
The `Type=Notify` support was added in this release cycle.  I've documented the dependency on libsystemd-dev, but I forgot to parametrize the smcroute.service.in file. Good catch, thanks! *fixing*